docs/admin/apparmor/index.md

```diff
@@ -13,15 +13,15 @@ added in Kubernetes v1.4.
 
 ## What is AppArmor
 
-AppArmor is a Linux kernel security module that supplements the standard Linux user and group based
-permissions to confine programs to a limited set of resources. AppArmor can be configured for any
+AppArmor is a Linux kernel security module that supplements the standard Linux user and group based 
+on permissions to confine programs to a limited set of resources. AppArmor can be configured for any
```

Comment:
the grammar reads : `AppArmor` is a `Linux kernel security module` that `supplements` `the standard Linux user and group based permissions`, but not based on